ipn/ipnserver/server.go

```diff
@@ -251,6 +251,17 @@ func (s *Server) checkConnIdentityLocked(ci *ipnauth.ConnIdentity) error {
 				return err
 			}
 
+			// https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-identifiers
```

Comment:
Can you run this URL through archive.org and use that instead? Microsoft doc URLs jump around a lot these days, so it would be nice to use something more permanent.

Reply:
done.